rwkv/rwkv_cpp_shared_library.py

```diff
@@ -83,11 +86,13 @@ def rwkv_init_from_file(self, model_file_path: str, thread_count: int, gpu_layer
         """
 
         ptr = self.library.rwkv_init_from_file(model_file_path.encode('utf-8'),
-                                               ctypes.c_uint32(thread_count),
-                                               ctypes.c_uint32(gpu_layers_count))
+                                               ctypes.c_uint32(thread_count))
         assert ptr is not None, 'rwkv_init_from_file failed, check stderr'
         return RWKVContext(ptr)
 
+    def rwkv_gpu_offload_layers(self, ctx: RWKVContext, gpu_layers_count: int) -> None:
+        assert self.library.rwkv_gpu_offload_layers(ctx.ptr, ctypes.c_uint32(gpu_layers_count)), 'rwkv_gpu_offload_layers failed, check stderr'
```

Comment:
Please add doc comment as in all other methods in the file (just copy-paste from header file, but in Python doc style)

Reply:
sorry, I was in visual studio at the time which does not support forwarding ctrl+k to `nano` for copy and paste, I will fix it in windows terminal